app/app.py

```diff
@@ -99,10 +98,14 @@ def run_recipe(recipe_name, request, fmt='json'):
 
 @app.route(API_BASE_PATH + "<recipe_name>")
 def run_recipe_api(recipe_name):
-    data = run_recipe(recipe_name, request)
-    result = make_response(data)
-    result.mimetype = 'application/json'
-    return result
+    try:
+        data = run_recipe(recipe_name, request)
+        result = make_response(data)
+        result.mimetype = 'application/json'
+        return result
+    except HTTPError as e:
```

Comment:
Maybe we could add a test that covers this? If it would take too long then feel free to ignore this.

Reply:
Thanks I added a test for the error.